pkgs/development/compilers/gcc/common/dependencies.nix

```diff
@@ -0,0 +1,92 @@
+{ lib
+, stdenv
+, version
+, buildPackages
+, targetPackages
+, texinfo
+, which
+, gettext
+, pkg-config ? null
+, gnused
```

Comment:
I wouldn't mind if we remove as a next step some of those ``? null``s.

Reply:
i think nulls are being used because of bootstrapping